networking/networking_operators/metallb-operator/metallb-upgrading-operator.adoc

```diff
@@ -6,37 +6,18 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-If you are currently running version 4.10 or an earlier version of the MetalLB Operator, please note that automatic updates to any version later than 4.10 do not work. Upgrading to a newer version from any version of the MetalLB Operator that is 4.11 or later is successful. For example, upgrading from version 4.12 to version 4.13 will occur smoothly.
+// Is the relationship with OCP updates correct? What about z-streams?
+A `Subscription` custom resource (CR) that subscribes the namespace to `metallb-system` by default sets the `installPlanApproval` parameter to `Automatic`. This means that when you update to the latest version of {product-title}, the MetalLB Operator is automatically upgraded to the latst version of the Operator.
```

Comment:
It is if the catalog to which subscription points to has a later version of operator, it will be automatically upgraded. If the catalog does not have newer version of operator, post upgrade of the cluster no changes occur.
Just stating every  z stream will not have new operator code. It is only when operator code is newer upgrade happen based on install plan approval strategy.

Reply:
Thanks, @asood-rh . I updated this paragraph to state:

> A `Subscription` custom resource (CR) that subscribes the namespace to `metallb-system` by default sets the `installPlanApproval` parameter to `Automatic`. This means that when the Red{nbsp}Hat-provided Operator catalogs includes a newer version of the MetalLB Operator, the MetalLB  Operator update happens automatically.

Reply:
Why would a customer want to change the Operator updates to manual? They have a specific environment where an update to the Operator might bring about more configuration changes?